### PR TITLE
Fixes #34174 - Add content view ID to content-view component list

### PR DESCRIPTION
**Pull request by rverdile**
Test by creating content views and a composite content view. List the component content views with:

`hammer content-view component list --composite-content-view-id <id>`

### CODE REVIEW

File: lib/hammer_cli_katello/content_view_component.rb [hunk 1]
```diff
@@ -84,16 +84,16 @@ class ListCommand < HammerCLIKatello::ListCommand
       include OrganizationOptions
       include CompositeContentViewNameResolvable
       output do
-        field :id, _("Id")
+        field :id, _("Component Id")
         field :content_view_name, _("Name")
         field :version, _("Version")
+        field :content_view_id, _("Id")
```
Comment (sjha4) [inline lib/hammer_cli_katello/content_view_component.rb:90]: This works well. We should probably switch the order to show the content view Id first and component Id in this column instead..Thoughts?

Comment (rverdile) [reply lib/hammer_cli_katello/content_view_component.rb:90]: Sounds good. Should it also read "Content View Id" instead of "Id"?

Comment (sjha4) [reply lib/hammer_cli_katello/content_view_component.rb:90]: That would make it clearer too.. 👍🏼

**Comment by rverdile** [comment]
updated

**Review by sjha4 — APPROVED** [review]

**Comment by chris1984** [comment]
ACK

**Pull request merged**